### PR TITLE
[Feature] Add temp_scaled_logprobs and top_p_normalized_logprobs parameters for logits and logprobs post processing

### PR DESCRIPTION
**Pull request by ckl117**
develop [PR#3552](https://github.com/PaddlePaddle/FastDeploy/pull/3552)
This PR introduces two new parameters to enhance log-probability sampling flexibility:

**temp_scaled_logprobs**

1. Applies temperature scaling directly to log-probabilities.
3. Helps in controlling randomness during decoding without affecting other probability adjustments.

**top_p_normalized_logprobs**

1. Normalizes log-probabilities after nucleus (top-p) filtering.
2. Ensures probability mass consistency and stable sampling behavior.

Service layer functional support [PR#3523](https://github.com/PaddlePaddle/FastDeploy/pull/3523)

```
# clinet send req
response = client.chat.completions.create(
    model="default",
    messages=[
        {"role": "user", "content": "Hello, who are you"},
    ],
    max_tokens=7,
    temperature=0.7,
    top_p=0.8,
    logprobs=True,
    top_logprobs=0,
    extra_body={
        "temp_scaled_logprobs": True,       # default False
        "top_p_normalized_logprobs": True,  # default False
    },
    stream=True,
)
```

### CODE REVIEW

**Review by ming1753 — APPROVED** [review]
LGTM

**Review by Jiang-Jia-Jun — APPROVED** [review]
